pyrobosim/pyrobosim/data/test_world.yaml

```diff
@@ -188,18 +188,20 @@ objects:
     parent: table0
     pose:
       position:
-        x: 1.0
-        y: -0.5
+        x: 0.15
+        y: 0.0
       rotation_eul:
-        yaw: 40.5
+        yaw: -40.5
```

Comment:
this changed because the table is rotated -90 degrees. So technically to be equivalent, this needs to be an offset by 90 degrees, not a negation. But it's close enough here since the angle is close to 45 degrees hehe.